src/parser/toplevel.lisp

```diff
@@ -2090,7 +2093,16 @@ consume all attributes")))
                  :primary-note "unexpected form"
                  :highlight :end)))
 
-  (make-struct-field
-   :name (symbol-name (cst:raw (cst:first form)))
-   :type (parse-type (cst:second form) file)
-   :source (cst:source form)))
+
+  ;; (identifier type "docstring")
+  (let (docstring)
```

Comment:
Add a parse error if the third element in a field is not a string. Currently it would be silently discarded:

```
(define-struct Point
  (x Integer 5)
  (y Integer (+ 1 2 4))
```